docs/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Breaking change: System.Drawing.Common only supported on Windows"
+description: Learn about the .NET 6 breaking change where the System.Drawing.Common package is no longer support on non-Windows operating systems.
+ms.date: 07/27/2021
+---
+# System.Drawing.Common only supported on Windows
+
+The [System.Drawing.Common](https://www.nuget.org/packages/System.Drawing.Common/) NuGet package is now attributed as a Windows-specific library. The platform analyzer emits warning at compile time when compiling for non-Windows operating systems.
+
+On non-Windows operating systems, unless you set a run-time configuration switch, a <xref:System.TypeInitializationException> exception is thrown with <xref:System.PlatformNotSupportedException> as the inner exception.
+
+## Old behavior
+
+Prior to .NET 6, using the System.Drawing.Common package did not produce any compile-time warnings, and no run-time exceptions were thrown.
+
+## New behavior
+
+Starting in .NET 6, the platform analyzer emits compile-time warnings when referencing code is compiled for non-Windows operating systems. In addition, the following run-time exception is thrown unless you set a configuration option:
```

Comment:
Can we mention that in .NET 7 the plan is to no longer support this not even with the runtime config switch?

Reply:
The docs style guide says not to discuss future plans in documentation. @tdykstra @BillWagner How strictly do we adhere to this?

Reply:
I see. The reason why I think in this case would be valuable, it is because we want to encourage customers to move off System.Drawing.Common for xplat.